docs/changelog/5_x.rst

```diff
@@ -0,0 +1,343 @@
+
+With these new commands here's how we envision the developers using them to manage "feature" branches:
+
+1) Create a new "feature" VCS branch (a clone of the "main" branch) and a
+   corresponding "feature" EdgeDB branch.
+
+2) Work on the "feature" branch, add migrations, etc.
+
+3) When it is time to merge the feature work back into the main branch we want
+   to arrange things so that the "feature" branch is in a state that is a
+   simple fast-forward w.r.t the "main" branch.
+
+4) In order to achieve the above state we need to make sure "main" code branch
+   as well as EdgeDB branch are both up-to-date.
+
+5) Then we want to rebase the "feature" branch code on top of the "main"
+   branch code.
+
+6) After that we need to replicate the same rebase operation with the EdgeDB
+   branch. Our CLI tools may need to first clone the "main" branch with the
+   data into a "temp" branch. Then we can introspect the migration histories
+   of "temp" and "feature" branches so that we can establish where they
+   diverge. Take all the divergent migrations from the "feature" branch and
+   apply them to the "temp" branch. If the operation is successful, drop the
+   "feature" branch and rename "temp" to "feature". We now have successfully
+   rebased "feature" branch on top of "main".
+
+7) Since the state of "feature" is now a straightforward fast-forward w.r.t.
+   the "main" branch we can finally merge "feature" back into main in VCS and
+   then merge the EdgeDB branch as well (or rename "feature" EdgeDB branch
+   into "main", if the old branch is no longer needed).
+
+
+Update pgvector extension
+-------------------------
+
+A new HNSW (Hierarchical Navigable Small Worlds) index has been added to the ``pgvector`` extension. Just like IVFFlat indexes there are three flavors of HNSW corresponding to different operations:
+
+* ``ext::pgvector::hnsw_euclidean``
+* ``ext::pgvector::hnsw_ip``
+* ``ext::pgvector::hnsw_cosine``
+
+We have also updated the mechanism for tuning all of the indexes provided in this extension. The ``probes`` (for IVFFlat) and ``ef_search`` (for HNSW) parameters can now be accessed via ``ext::pgvector::Config`` object.
+
+The current config value can be found using this query:
+
+.. code-block:: edgeql-repl
+
+    db> select ext::pgvector::Config.probes;
+    {1}
+
+Updating the value can be done by ``configure session`` command:
+
+.. code-block:: edgeql-repl
+
+    db> configure session
+    ... set ext::pgvector::Config::probes := 5;
+    OK: CONFIGURE SESSION
+
+It is also possible to restore the default config value:
+
+.. code-block:: edgeql-repl
+
+    db> configure session reset ext::pgvector::Config::probes;
+    OK: CONFIGURE SESSION
+
+
+Authentication
+--------------
+
+We've expanded our ``auth`` extension with "WebAuthn" and "Magic Links" functionality.
```

Comment:
Hey, @scotttrinh. Is there more we should say here?

Reply:
Yep.

```suggestion
We're bringing two popular "passwordless" authentication schemes to our ``auth`` extension: the Web Authentication API (commonly known as WebAuthn or Passkeys), as well as email-based "magic links".

We've also added two popular chat platforms to our list of supported OAuth providers: Slack and Discord.
```

---

We're also missing these two PRs:

- https://github.com/edgedb/edgedb/pull/6708
- https://github.com/edgedb/edgedb/pull/6720

Reply:
I was thinking that while I had only the brief details in this section I would include PRs (so that there's some indicator of what goes there), but typically we only include PRs in the "Additional Changes" where most of them are mentioned rather tersely. In these "dedicated" sections we tend to just have text descriptions and examples rather than PR links.